07/ID_funkce/ID_format.py

```diff
@@ -0,0 +1,11 @@
+def ID_format(ID):
+
+    first_six_numbers = ID[:6]
+    slash = ID[6]
+    last_four_numbers = ID[7:]
```

Comment:
Luci, tu stále potenciálne môže vzniknúť ten problém, ktorý som ti písala - ty chceš všetko od 8. znaku do konca ale ak bude mať vstup formát napr. 123456/1 tak to prejde. Takže tu je potreba si overiť, že to sú 4 znaky.

Reply:
jasné, to mi uniklo...